documentation/man/features/umount.rst

```diff
@@ -12,14 +12,13 @@ DESCRIPTION
 ===========
 Projects that use a QEMU VM in the development cycle to host the development
 environment usually require that developers use some specific commands to
-manage this VM. This explains the `umount` command available in **kw**.
+manage this VM. This explains the **umount** command available in **kw**.
```

Comment:
How about rewrite this part? Maybe something like this:

[..] some specific commands to manage the access to the VM files. For this reason, we have commands like umount and mont.